tests/test_assets.py

```diff
@@ -450,16 +457,41 @@ def test_lookup_symbol(self):
                 self.assertEqual(result.symbol, 'EXISTING')
                 self.assertEqual(result.sid, i)
 
-    @parameterized.expand(
-        build_lookup_generic_cases()
-    )
-    def test_lookup_generic(self, finder, symbols, reference_date, expected):
+    def test_lookup_symbol_from_multiple_valid(self):
```

Comment:
Yes, there are several examples of this - HK, AGN, and ES, and others. These are symbols which multiple companies share and which have overlapping date ranges. The current logic in lookup_symbol dealt with such cases by sorting the multiple valid equities by start/end date and returning the latest, so I was just trying to mimic that logic in my cached version. 

Should I change anything?